CE: persister should include messages at timestamp (#413)
vaadin/docs: articles/ce/collaboration-message-list.asciidoc

When a new message is submitted, the data flows through your backend.
First, the message list saves the message to your backend with the persister.
Then, the message list makes a query to the persister to fetch all the messages with a timestamp
that is equal or more recent than the last message in memory.
The new messages are again stored in the Collaboration Engine's memory,
updating all the message list instances which are connected to the same topic.

> It's important to note when implementing the `fetchMessages` method, that the returned

Reviewer: ```suggestion
When implementing the [methodname]#fetchMessages()# method, it is important to note that the returned
```

> stream should include the messages sent exactly at the `FetchQuery::getSince` time.

Reviewer: ```suggestion
stream should include the messages sent exactly at the time that you can get with [methodname]#getSince()# from the [classname]#FetchQuery#.
```
